bin/vl2vg

```diff
@@ -9,7 +9,7 @@ var helpText =
 
 // import required libraries
 var fs = require('fs'),
-    vl = require('../vega-lite.js');
+    vl = require('vega-lite');
```

Comment:
Unfortunately this breaks our image build script. We call vl2vg from package.json on `npm run build:images`. So this won't work unless we install vega lite globally. I'm happy to accept the change below but cannot merge unless we can call the bin scripts from local as well.